koku/reporting_common/models.py

```diff
@@ -68,10 +88,15 @@ class Meta:
     completed_datetime = models.DateTimeField(null=True)
     started_datetime = models.DateTimeField(null=True)
     etag = models.CharField(max_length=64, null=True)
+    # id of the task processing this report
+    celery_task_id = models.UUIDField(null=True)
+    # Current status of processing report
+    status = models.IntegerField(choices=STATUS_CHOICES, default=STATUS_DOWNLOADING)
+    failed_status = models.IntegerField(choices=FAILED_STATUS_CHOICES, null=True)
```

Comment:
I'm not super stoked on the readability of the failed status here tbh.

Reply:
Yup, there's room for improvement there.

Reply:
It could just be a separate class.
```python
class FailedStatus(IntegerChoices):
     FAILED = Status.FAILED
     DOWNLOADING = Status.DOWNLOADING
     OCP_CLOUD_PROCESSING = (Status.OCP_CLOUD_PROCESSING, "OCP-Cloud-Processing")
```

Reply:
Yeh sorry I'm slow today! many meetings :face_exhaling: I was planning to do exactly this. Hopefully I'll get it done by tomorrow morning. Thanks again for the help. :+1: